bundles/best_practices.rst

```diff
@@ -166,6 +166,69 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+    env:
+        global:
+            - TEST_COMMAND="phpunit"
```

Comment:
Using `phpunit` as test command on Travis tends to create issue depending on your supported PHP versions: PHP 7 comes with PHPUnit 6, while PHP 5.3 images (for bundles still supporting it) come with a 4.8 version *before* the introduction of the namespaced classes.